src/constants.js

```diff
@@ -21,14 +21,9 @@
 // SAMPLED_MASK is the bit mask indicating that a span has been sampled.
 export const SAMPLED_MASK = 0x1;
 
-// DEBUG_MASK is the bit mask indicationg that a span has been marked for debug.
+// DEBUG_MASK is the bit mask indicating that a span has been marked for debug.
 export const DEBUG_MASK = 0x2;
 
-// DEFERRED_SAMPLING_MASK is the bit mask indicating that the upstream service that generated the
-// span did not make a definitive sampling decision, but deferred it to be made by some
-// downstream service.
-export const DEFERRED_SAMPLING_MASK = 0x4;
```

Comment:
to keep a cleaner git history, can you revert deferred sampling diff first in master, and rebase this? I would rather see diffs vs. "finalized" logic, not the deferred logic that we're throwing out.